### PR TITLE
[interop-ui-tests] update scenarios

### PR DESCRIPTION
**Pull request by shvasude**
Jira Id: https://issues.redhat.com/browse/HELM-15
Bugzill ID: Bug [1922907](https://bugzilla.redhat.com/show_bug.cgi?id=1922907)
Update steps for feature:
install-HelmChart
helm-Installation-View
actions-on-HelmRelease
topology-HelmRelease
helm-Navigation

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature [hunk 1]
```diff
@@ -2,36 +2,39 @@ Feature: Helm Chart Installation View
...
+    Scenario: Switch from YAML to Form view
+        Given user is at the Install Helm Chart page
+        When user selects the YAML View
+        And user does some changes in the yaml for helm chart
+        And user selects the Form view
+        And user comes back to YAML view
+        Then user will see that the data hasn't lost
+
+
+    @regression, @smoke, @manual
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature:32]: As discussed, please remove the @regression tag when both [@regression, @smoke,] are on scenarios

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature [hunk 2]
```diff
@@ -2,36 +2,39 @@ Feature: Helm Chart Installation View
...
+    Scenario: Switch from Form to YAML view
+        Given user is at the Install Helm Chart page
+        When user selects the Form View
+        And user will see values and choices are pulled from JSON Schema associated with chart
+        And user does some changes in the form for helm chart
+        And user selects the YAML view
+        And user comes back to Form view
+        Then user will see that the data hasn't lost
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature:40]: Add empty lineto avoid lint issues

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature [hunk 3]
```diff
@@ -1,129 +1,123 @@
...
-
-Scenario: Actions menu of Helm Details page : P-03-TC10
-   Given user is at the Helm page
-   When user clicks Actions menu in Helm Details page
-   Then Actions menu display with options Upgrade, Rollback, and Uninstall Helm Release
+    Background:
+        Given user is at developer perspective
+        And user has selected namespace "aut-helm-navigation"
+
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature:8]: As discussed, please remove the @regression tag when both [@regression, @smoke,] are on scenarios

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature [hunk 4]
```diff
@@ -1,129 +1,123 @@
...
+    Scenario: Open the Helm tab on the navigation bar when helm charts are absent: HR-11-TC02
+        Given user is at developer perspective
+        When user clicks on the Helm tab
+        Then user will be redirected to Helm releases page
+        And user is able to see the message as no helm charts present
+        And user will get the link to install helm charts from developer catalog
+
+
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature:17]: As discussed, please remove the @regression tag when both [@regression, @smoke,] are on scenarios

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature [hunk 5]
```diff
@@ -1,129 +1,123 @@
...
+        Given user is at Add page
+        When user selects "Helm Chart" card from add page
+        And user searches "Nodejs Ex K v0.2.1" card from catalog page
+        And user selects "Nodejs Ex K v0.2.1" helm chart from catalog page
+        And user clicks on the Install Helm Chart button on side bar
+        Then Install Helm Chart page is displayed
+        And release name displays as "nodejs-ex-k"
+
+
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature:28]: As discussed, please remove the @regression tag when both [@regression, @smoke,] are on scenarios

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature [hunk 6]
```diff
@@ -1,129 +1,123 @@
...
+    Scenario: Yaml view editor for Install Helm Chart page: HR-02-TC05
+        Given user is at Install Helm Chart page
+        When user selects YAML view
+        Then user is able to see YAML editor
+
+
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature:35]:
```suggestion
    @smoke
```

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature [hunk 7]
```diff
@@ -1,129 +1,123 @@
...
+        When user selects "Helm Chart" card from add page
+        And user searches "Nodejs Ex K v0.2.1" card from catalog page
+        And user selects "Nodejs Ex K v0.2.1" helm chart from catalog page
+        And user clicks on the Install Helm Chart button on side bar
+        And user clicks on the Install button
+        Then user will be redirected to Topology page
+        And Topology page have the helm chart workload "nodejs-example"
+
+
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature:47]:
```suggestion
    @smoke
```

File: frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature [hunk 8]
```diff
@@ -12,12 +12,12 @@ Feature: Install the Helm Release
         Then user can see "Helm Chart" card on the Add page
 
 
-    @regression, @smoke, @manual
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature:15]: As discussed, please remove the @regression tag when both [@regression, @smoke,] are on scenarios

File: frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature [hunk 9]
```diff
@@ -1,71 +1,69 @@
...
-    When user clicks on the Actions drop down menu
-    Then user will see the "Upgrade" action item
-    And user will see the "Rollback" action item
-    And user will see the "Uninstall Helm Release" action item
+    Background:
+        Given user is at developer perspective
+        And user has selected namespace "aut-helm-sidebar"
+        And helm release "nodejs-ex-k" is present in topology page
+
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature:9]: As discussed, please remove the @regression tag when both [@regression, @smoke,] are on scenarios

File: frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature [hunk 10]
```diff
@@ -1,71 +1,69 @@
...
+    Scenario: Context menu options of helm release: HR-07-TC01
+        When user right clicks on the helm release "nodejs-ex-k" to open the context menu
+        Then user is able to see the context menu with actions Upgrade, Rollback and Uninstall Helm Release
+
+
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature:15]:
```suggestion
    @smoke
```

File: frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature [hunk 11]
```diff
@@ -1,71 +1,69 @@
...
+
+    @regression
+    Scenario: Routes link on the sidebar for the Helm Release: HR-10-TC07
+        Given user is on the topology sidebar of the helm release "nodejs-ex-k"
+        When user switches to the "Resources" tab
+        And user clicks on the link for the "routes" of helm release
+        Then user is redirected to the "routes" Details page for the helm release
+
+
+    @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature:63]:
```suggestion
    @smoke
```

File: frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature [hunk 12]
```diff
@@ -1,71 +1,69 @@
...
+    Scenario: Actions drop down on the side bar: HR-10-TC08
+        Given user is on the topology sidebar of the helm release "nodejs-ex-k"
+        When user clicks on the Actions drop down menu
+        Then user will see the "Upgrade" action item
+        And user will see the "Rollback" action item
+        And user will see the "Uninstall Helm Release" action item
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature:69]: Add empty line at eof

**Review by makambalaji — COMMENTED** [review]
After completion, please assign back to me

File: frontend/packages/dev-console/integration-tests/features/helm/actions-on-HelmRelease.feature [hunk 13]
```diff
@@ -1,58 +1,58 @@
...
+
+
+    @regression, @smoke
+    Scenario: Uninstall Helm Release through Context Menu: HR-08-TC04
+        Given user is at the Topology page
+        When user right clicks on the helm release "nodejs-ex-k" to open the context menu
+        And user clicks on the "Uninstall Helm Release" action
+        And user enters the release name
+        And user clicks on the Uninstall button
+        Then user will be redirected to Topology page with no workloads
```
[the remark this answers is not in the document]
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/actions-on-HelmRelease.feature]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature [hunk 14]
```diff
@@ -2,36 +2,39 @@ Feature: Helm Chart Installation View
...
+        When user selects "From Catalog" card from add page
+        And user selects "Helm Charts" option from Type section
+        And user searches 'Nodejs Ex K v0.2.1' card from catalog page
+        And user selects "Nodejs Ex K v0.2.1" helm chart from catalog page
+        And user will see the information of all the chart versions together
+        And user clicks on the Install Helm Chart button on side bar
+        Then user will see the chart version dropdown
+
+
+    @regression, @smoke, @manual
```
[the remark this answers is not in the document]
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature:22]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature (hunk 1 above)
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/helm-Installation-View.feature:32]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

File: frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature (hunk 3 above)
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/helm-Navigation.feature]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

File: frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature (hunk 8 above)
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

File: frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature (hunk 9 above)
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

File: frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature (hunk 10 above)
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

File: frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature (hunk 11 above)
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

File: frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature (hunk 12 above)
Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/topology-HelmRelease.feature]: done, 73c363ec6787648fc93c8143b395c02f13b28abc

**Pull request merged**